Packages/com.unity.inputsystem/InputSystem/Devices/Commands/QueryCanRunInBackground.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Runtime.InteropServices;
+using UnityEngine.Experimental.Input.Utilities;
+
+namespace UnityEngine.Experimental.Input.LowLevel
+{
+    [StructLayout(LayoutKind.Explicit, Size = InputDeviceCommand.kBaseCommandSize + sizeof(bool))]
+    internal unsafe struct QueryCanRunInBackground : IInputDeviceCommandInfo
```

Comment:
If it's Query Capabilities it would change how it's used.
Right now I call it when I care about if it can run in background, direct from the property, but if I'm filling in a whole bunch of settings it should be more of a query on connect and cache.